pkgs/development/python-modules/uvicorn/default.nix

```diff
@@ -35,6 +35,11 @@ buildPythonPackage rec {
 
   patches = [ ./setup.patch ];
 
+  # LICENCE.md gets propagated without this, causing collisions
+  postInstall = ''
+    rm $out/LICENSE.md
+  '';
```

Comment:
@FRidh do you know why the LICENSE.md files get copied to the top-level like that?

Reply:
It seems to be [listed](https://github.com/encode/uvicorn/blob/141a817d66bfa3c5a3d0810fd2fa6a239f5f187e/setup.py#L64) in Uvicorn's `setup.py` as a `data_file` so [this happens](https://docs.python.org/2/distutils/setupscript.html#installing-additional-files). Somebody has already filed an [issue](https://github.com/encode/uvicorn/issues/392).

Reply:
Thanks for the explanation. It's curious that `requests-async` and `httpcore` also suffer from the same issue (both in this PR as well). But as long as it's not a widespread issue I guess it's ok to leave the postInstall here.